doc/developer-guide.md

```diff
@@ -418,6 +418,10 @@ cmake llvm -GNinja \
   -DLLVM_TARGETS_TO_BUILD='X86;ARM;AArch64'
 ```
 
+Note that `RISCV` will need to be added to `-DLLVM_TARGETS_TO_BUILD` for any
+RISC-V target, and `-DLLVM_ENABLE_PROJECTS=clang;lld` will be required for most
+non "host" targets e.g. `refsi`.
```

Comment:
Given that the instructions are not meant to be optimal, merely meant to work, maybe we should just have the instructions say `-DLLVM_ENABLE_PROJECTS='clang;lld' -DLLVM_TARGETS_TO_BUILD='X86;ARM;AArch64;RISCV`. This should work in all cases. We can then add a note saying that lld is only needed in some cases, and it's not necessary to build all targets, but it still gives a command that people can just copy and paste.

Reply:
I think that makes sense, yes